src/router/index.ts

```diff
@@ -15,3 +17,7 @@ router.use('/issue', withAwsAuth, issueRouter);
 router.use('/projects', withAwsAuth, projectRouter);
 router.use('/roles', withAwsAuth, roleRouter);
 router.use('/issue-providers', withAwsAuth, issueProviderRouter);
+
+if (Constants.NODE_ENV === NodeEnv.DEV) {
+  router.use('/ngrok', ngrokHealthRouter);
```

Comment:
This is gonna respond with OK code every time the project is running, independently if Ngrok is up and running or not

Reply:
You can do a /health endpoint to check server health

Reply:
solved